glean-core/ios/Glean.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		1F60589723148BF800307A9F /* GleanLifecycleObserver.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F60589623148BF800307A9F /* GleanLifecycleObserver.swift */; };
```

Comment:
I'm already getting annoyed by the fact that every PR will change the project file now ... (not your fault of course)

Reply:
Yeah, it's annoying to be sure.  It's definitely not as clean as the Android approach to this.